src/components/SearchResults.jsx

```diff
@@ -0,0 +1,104 @@
+import React, { useState } from "react";
+
+import Bookings from "./Bookings";
+import CustomerProfile from "./CustomerProfile";
+import TableRow from "./TableRow";
+function SearchResults({ results }) {
+  const [customerId, setCustomerId] = useState(null);
+  const sortedResults = [...results];
```

Comment:
`sortedResults` is being mutated by the `sort` method so beware of calling a mutating method on this array